.github/check-additional-targets.yml

```diff
@@ -0,0 +1,45 @@
+name: Check_additional_targets
+
+on:
+  pull_request:
+    branches:
+     - master
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  check_compilation:
+    name: Check compilation
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        target: [aarch64-apple-ios, x86_64-apple-ios, aarch64-apple-darwin, x86_64-apple-darwin, aarch64-linux-android, armv7-linux-androideabi, i686-linux-android, x86_64-linux-android]
```

Comment:
I've added **every possible** Android and iOS target here, which gives the strongest guarantees about supported targets. If that's too much CI usage, some subset of this would probably get most of the way there too.

Reply:
I'm game to try this at least